app/views/application/_pickup_posts.html.haml

```diff
@@ -1,5 +1,5 @@
 - if pickup_posts.exists?
   %h2.subheading.subheading--aside.subheading--press-articles.subheading--press-articles--pickup ピックアップ記事
-  %ul
-    - pickup_posts.includes(post: [:categories]).ordered.each do |pickup_post|
+  %ul.articles--pickup
```

Comment:
`.pickup-articles` or `.articles.articles--pickup`

You shouldn't use modifier only.
Modifier is just like a "mix-in".